hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestFSImage.java

```diff
@@ -275,6 +276,29 @@ public void testSaveAndLoadStripedINodeFile() throws IOException{
     }
   }
 
+  @Test
+  public void testImportCheckpoint() {
+    Configuration conf = new Configuration();
+    conf.set(DFSConfigKeys.DFS_NAMENODE_CHECKPOINT_EDITS_DIR_KEY, "");
+    MiniDFSCluster cluster = null;
+    try {
+      cluster = new MiniDFSCluster.Builder(conf).build();
+      cluster.waitActive();
+      FSNamesystem fsn = cluster.getNamesystem();
+      FSImage fsImage= new FSImage(conf);
+      fsImage.doImportCheckpoint(fsn);
+      fail("Expect to throw IOException.");
+    } catch (IOException e) {
+      GenericTestUtils.assertExceptionContains(
+              "Cannot import image from a checkpoint. "
+                      + "\"dfs.namenode.checkpoint.edits.dir\" is not set.", e);
```

Comment:
Use LambdaTestUtils instead of try-catch-assert

Reply:
Thanks @ayushtkn for your advice, just updated.